IsRebalancingEnabled blocking call

### Describe the bug

When retrieving rest info this invokes a blocking call in the netty event loop thread.

```
10:34:16,164 ERROR (non-blocking-thread-RestMergeTest-NodeB-3-5:[]) [o.i.c.t.TestSuiteProgress] Test failed: org.infinispan.rest.partition.RestMergeTest[DIST_SYNC, DENY_READ_WRITES].BlockingChecker
java.lang.AssertionError: Blocking call! jdk.internal.misc.Unsafe#park on thread Thread[#123,non-blocking-thread-RestMergeTest-NodeB-3-5,5,ISPN-non-blocking-group]
	at org.infinispan.util.CoreTestBlockHoundIntegration.lambda$applyTo$0(CoreTestBlockHoundIntegration.java:52)
	at reactor.blockhound.BlockHound$Builder.lambda$install$8(BlockHound.java:488)
	at reactor.blockhound.BlockHoundRuntime.checkBlocking(BlockHoundRuntime.java:89)
	at java.base/jdk.internal.misc.Unsafe.park(Unsafe.java)
	at java.base/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:269)
	at java.base/java.util.concurrent.CompletableFuture$Signaller.block(CompletableFuture.java:1910)
	at java.base/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:4021)
	at java.base/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3967)
	at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1983)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2139)
	at org.infinispan.commons.util.concurrent.CompletableFutures.await(CompletableFutures.java:122)
	at org.infinispan.commons.util.concurrent.CompletionStages.join(CompletionStages.java:89)
	at org.infinispan.topology.LocalTopologyManagerImpl.isCacheRebalancingEnabled(LocalTopologyManagerImpl.java:778)
	at org.infinispan.topology.LocalTopologyManagerImpl.isRebalancingEnabled(LocalTopologyManagerImpl.java:766)
	at org.infinispan.manager.CacheManagerInfo.isRebalancingEnabled(CacheManagerInfo.java:178)
	at org.infinispan.manager.CacheManagerInfo.toJson(CacheManagerInfo.java:276)
	at org.infinispan.rest.resources.ContainerResource.getInfo(ContainerResource.java:200)
	at org.infinispan.rest.framework.impl.RestDispatcherImpl.dispatch(RestDispatcherImpl.java:99)
	at org.infinispan.rest.RestRequestHandler.handleRestRequest(RestRequestHandler.java:208)
	at org.infinispan.rest.RestRequestHandler.channelRead0(RestRequestHandler.java:158)
	at org.infinispan.rest.RestRequestHandler.channelRead0(RestRequestHandler.java:54)
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)
	at io.netty.handler.codec.http.HttpServerKeepAliveHandler.channelRead(HttpServerKeepAliveHandler.java:64)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at org.infinispan.rest.CorsHandler.channelRead(CorsHandler.java:99)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
```

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### Infinispan version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java
+++ b/server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java
@@ -197,8 +197,11 @@ public class ContainerResource implements ResourceHandler {
       if (responseBuilder.getHttpStatus() == NOT_FOUND) return completedFuture(responseBuilder.build());
 
       EmbeddedCacheManager cacheManager = invocationHelper.getRestCacheManager().getInstance();
-      Json cacheManagerInfo = cacheManager.getCacheManagerInfo().toJson();
-      return asJsonResponseFuture(invocationHelper.newResponse(request), cacheManagerInfo, isPretty(request));
+      return CompletableFuture.supplyAsync(() -> {
+         // toJson is blocking as it invokes the command on the coordinator
+         Json cacheManagerInfo = cacheManager.getCacheManagerInfo().toJson();
+         return addEntityAsJson(cacheManagerInfo, responseBuilder, isPretty(request)).build();
+      }, invocationHelper.getExecutor());
    }
 
    private CompletionStage<RestResponse> setRebalancing(boolean enable, RestRequest request) {

